docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
+
+services:
+  veni:
+    container_name: FFXIVVenues.Veni
+    image: ffxivvenues/ffxivvenues.veni:latest
+    restart: unless-stopped
+    depends_on:
+      - mongo
+    networks:
+      - FFXIVVenues.Network
+      - FFXIVVenues.Veni.Network
+    environment:
+      - FFXIVVENUES_VENI__Persistence__Provider=MongoDb
+      - FFXIVVENUES_VENI__Persistence__Provider=mongodb://FFXIVVenues.Veni.Mongo:27017
```

Comment:
Need to check if this is loading in the other environment variables that aren't listed here from `.env`. I think I've checked already but will check again.

Reply:
Removed whole file now, it lives in the infrastructure repo now